manifests/script-lab-edge.xml

```diff
@@ -17,7 +17,11 @@
     <AppDomain>https://github.com</AppDomain>
     <AppDomain>https://stackoverflow.com</AppDomain>
     <AppDomain>https://dev.office.com</AppDomain>
-    <AppDomain>https://bornholm-runner-edge.azurewebsites.net</AppDomain>  
+    <AppDomain>https://script-lab.azureedge.net</AppDomain>
+    <AppDomain>https://script-lab-runner.azureedge.net</AppDomain>
+    <AppDomain>https://bornholm-runner-edge.azurewebsites.net</AppDomain>
```

Comment:
Being in app domains vs. being available on the UI are two completely different things.  App Domains just define the _whitelist_ of URLs that are going to be allowed to navigate to.   There is nothing wrong with defining more that we'll ever navigate to in practice.  So I think it's perfectly fine to add the localhost ones for all, just for purposes of keeping them as in-sync as possible for ease of diffing & copy-paste-ing.